preparing-the-vm.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+
+Complete the following steps to run Kite example code on a Cloudera Quickstart VM.
+
+* Install a VirtualBox or VMWare [Cloudera QuickStart VM][getvm] version 5.2 or later.
+* In that VM, run the following command from a terminal window. This command clones a local copy of Kite code examples you can build and run.
+
+```bash
+git clone https://github.com/kite-sdk/kite-examples.git
+```
+* Download the `kite-dataset` CLI JAR. To install the Kite CLI using `curl` and give execution permission to `kite-dataset.jar`, run the following commands from a terminal window.
```

Comment:
It would be better not to require the user to update the version of kite-dataset in the VM. Is there something not working that requires this update?